core/src/main/java/com/sequenceiq/cloudbreak/converter/stack/cluster/gateway/ClusterRequestToGatewayConverter.java

```diff
@@ -45,31 +53,12 @@ private void setBasicProperties(GatewayJson source, Gateway gateway) {
 
     private void setTopologies(GatewayJson source, Gateway gateway, boolean legacyGatewayRequest) {
```

Comment:
This could be moved to GatewayConverterUtil too

Reply:
Also when it is moved, the legacyGatewayRequest parameter can be dropped because the GatewayConverterUtil can decide whether it is a legacy request or not